pkg/common/v1alpha2/katib_manager_util.go

```diff
@@ -87,3 +87,14 @@ func DeleteExperiment(request *api_pb.DeleteExperimentRequest) (*api_pb.DeleteEx
 	kc := kcc.KatibManagerClient
 	return kc.DeleteExperiment(ctx, request)
 }
+
+func RegisterTrial(request *api_pb.RegisterTrialRequest) (*api_pb.RegisterTrialReply, error) {
+	ctx := context.Background()
+	kcc, err := getKatibManagerClientAndConn()
+	if err != nil {
+		return nil, err
+	}
+	defer kcc.Conn.Close()
```

Comment:
It is a little hack here. Could we define a func in katibManagerClientAndConn to close the conn?